README.md

```diff
@@ -4,34 +4,35 @@
 
 While Russia is brutally attacking our brothers and sisters in Ukraine, it is almost impossible for Russian citizens to get accurate information about these events. While it is simple for Russian ISPs, under pressure from their government, to block certain news sources from outside Russia it is much more difficult to block the whole internet.
 
-We created this simple script to enable all website owners to display informational messages about the ongoing events to visitors from a Russian IP address. We had a few additional goals:
+We created this script to enable all website owners to display informational messages about the ongoing events to visitors from a Russian IP address. We had a few additional goals:
 
 - It needed to be very lightweight.
 - It must not rely on a single domain name that would be easy to block.
-- It should not contain any tracker or other fancy abuse of personal data.
+- It should not contain any tracker or abuse of personal data.
 - Messages should be included in a single small footprint file.
 
 We believe we achieved all these goals.
 
 ## What?
-The setup is very simple. The msg.js file is called when the webpage is loaded. It will perform a call to a cloudflare worker instance that verifies if the IP address is within the Russian IP space. If so, a small banner in the Ukrainian national colors will be displayed at the top of the page with a message about the current events.
 
-You're free to remove or add any messages you want. We are not responsible for the content you wish to show on your own webpage :)
+The setup is very simple. When the webpage is loaded, it will perform a call to a cloudflare worker instance that verifies if the IP address is within the Russian or Belarussian IP space. If so, a small banner in the Ukrainian national colors will be displayed at the top of the page with a message about the current events.
 
-We are also releasing the uaware-worker for those that want to run their own worker. You can find it [here](https://github.com/UAware/UAware-worker)
+We are also releasing the `UAware-worker` for those that want to run their own worker. You can find it here: [UAware-worker](https://github.com/UAware/UAware-worker)
 
 ## How?
-You can use the msg.js file as is. We would recommend that you rename it to a random filename but that is not required. When using the standard file, this is what you would add to your webpage (replace js/ with the correct location of your javascript assets) :
 
-`<script type="text/javascript" src="js/msg.js"></script>`
+Add the following script tag to your HTML. It can go either in the `head` tag or at the end of the `html` tag. Either way will work fine.
+
+`<script type="text/javascript" src="https://cdn.skypack.dev/uaware-javascript" defer></script>`
```

Comment:
In the original version, the philosophy was that we had a script that people could use to include on their website from their own server. This was to make sure there wouldn't be a way to block the script from loading from other sources.

With the way the this refactoring has been done, is it possible to provide a script (we can add it to releases in this repository) that people can just download from this repository to include in their pages instead of just including the URL from skypack?

Reply:
Good point! I'll have a think about this

Reply:
OK, I've now gone back to the single file download approach, but using github releases this time.